robottelo/ui/contentviews.py

```diff
@@ -540,12 +540,9 @@ def fetch_puppet_module(self, cv_name, module_name):
     def copy_view(self, name, new_name):
         """Copies an existing Content View."""
         self.search_and_click(name)
-        self.edit_entity(
-            locators['contentviews.copy'],
-            locators['contentviews.copy_name'],
-            new_name,
-            locators['ak.copy_create']
-        )
+        self.perform_entity_action('Copy')
+        self.assign_value(locators['ak.copy_name'], new_name)
```

Comment:
`ak.copy_name`? In case it is expected that `ak.copy_name` is equal to `contentviews.copy_name` we can move locator to common module entirely